src/todo/ToDoContainer.js

```diff
@@ -12,10 +12,15 @@ function TodoContainer(){
         ]);
     };
 
+    const onDeleteTodoClick = text => {
```

Comment:
I would switch the names, so `onMyEvent` would be a prop, and `myEvent` would be the function that handles it